pkg/channel/distributed/common/config/secretwatcher.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2020 The Knative Authors
+
```

Comment:
I think they want these to be current 2021 - check all new files.

Reply:
Updated to 2021 for the current fileset.